### PR TITLE
JS fundamentals workshop 1 completed

### PR DESCRIPTION
**Pull request by ChanWinnie**

### CODE REVIEW

File: __workshop/exercise-2.md [hunk 1]
```diff
@@ -109,7 +127,14 @@ _THIS IS SUPER IMPORTANT_, otherwise Prettier might destroy your solution:
 7. }
 
 ```
-// Your solution here!
+6. let count = 10;
+3. let total = 0;
+
+2.     while (count > 0) {
+1.          total = total + count;
+5.          console.log('The total is:', total)
```
Comment (orangesmartie92) [inline __workshop/exercise-2.md:135]: 5 should be the last one (outside of the while loop!

**Review by orangesmartie92 — CHANGES_REQUESTED** [review]
exercise 10 failing

**Comment by orangesmartie92** [comment]
please redo question 10!

**Comment by ChanWinnie** [comment]
Done!

________________________________

**Comment by orangesmartie92** [comment]
ok! all working. closing.

**Pull request closed without merging**